pkg/sdn/registry/hostsubnet/rest.go

```diff
@@ -0,0 +1,66 @@
+package hostsubnet
+
+import (
+	"fmt"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/registry/generic"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/fielderrors"
+
+	"github.com/openshift/origin/pkg/sdn/api"
+	"github.com/openshift/origin/pkg/sdn/api/validation"
+)
+
+// sdnStrategy implements behavior for HostSubnets
+type sdnStrategy struct {
+	runtime.ObjectTyper
+}
+
+// Strategy is the default logic that applies when creating and updating HostSubnet
+// objects via the REST API.
+var Strategy = sdnStrategy{kapi.Scheme}
+
+func (sdnStrategy) PrepareForUpdate(obj, old runtime.Object) {}
+
+// NamespaceScoped is false for sdns
+func (sdnStrategy) NamespaceScoped() bool {
+	return false
+}
+
+func (sdnStrategy) GenerateName(base string) string {
+	return base
+}
+
+func (sdnStrategy) PrepareForCreate(obj runtime.Object) {
+}
+
+// Validate validates a new sdn
+func (sdnStrategy) Validate(ctx kapi.Context, obj runtime.Object) fielderrors.ValidationErrorList {
+	hs := obj.(*api.HostSubnet)
+	return validation.ValidateHostSubnet(hs)
+}
+
+// AllowCreateOnUpdate is false for sdns
+func (sdnStrategy) AllowCreateOnUpdate() bool {
+	return false
+}
+
+// ValidateUpdate is the default update validation for a HostSubnet
+func (sdnStrategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) fielderrors.ValidationErrorList {
+	return validation.ValidateHostSubnet(obj.(*api.HostSubnet))
```

Comment:
typically a ValidateUpdate method takes the old and the new object to know if the fields changed can be changed.  

see: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/validation/validation.go#L1378 as an example.

What happens if I update any of these fields?  Does my entire network update?  What happens to running pods?